contracts/CompoundHandler.sol

```diff
@@ -113,6 +121,14 @@ contract CompoundHandler is Handler, ReentrancyGuard {
             "deposit: Fail to supply to compound!"
         );
 
+        uint256 compBalance = IERC20(compAddress).balanceOf(address(this));
```

Comment:
Can define a common function, something like `transferCompOut()`, to avoid duplicate code. Also, should we have a threshold when transfer out?

Reply:
@joyious  Okay, I will define a new function, how about `claimComp()`?

Reply:
> Also, should we have a threshold when transfer out?

I am not sure, I think the COMP should be distributed in time, had better not wait for some time, what do you think? @bamboo-seven